### ISSUE TITLE
Deployment on kubernetes with locan-env services Pending forever

### ISSUE THREAD

**Issue by crlsgms** [wazuh/wazuh-kubernetes#660]
after raw initial deployment with .git package, services never get properly setup, any clues on fixing this, or to create proper ingresses?

I'm using 4.7.3, as newer images are no more available to fetch, even the VERSION is set to 5.0.0 on the VERSION file

![image](https://github.com/wazuh/wazuh-kubernetes/assets/4990453/8e0dc3da-9c7b-4171-a141-86fd241e16f6)

**Comment by vcerenu** [comment]
Hello @crlsgms

To deploy Wazuh in Kubernetes, I recommend that you use our documentation:
https://documentation.wazuh.com/current/deployment-options/deploying-with-kubernetes/index.html

Within the documentation, it is clarified that you have to use the latest release, which in this case is `v4.7.3`.
If you are deploying to Kubernetes and trying to pull images from Wazuh `5.0.0` then you are using the `master` branch, which contains our latest version in development, I recommend that you delete the entire stack and redeploy with the `v4.7.3` tag so that the deployment is correct.
I also recommend that you update the `Storage Class` with the driver corresponding to your local version of Kubernetes.

**Comment by crlsgms** [comment]
Thanks for the attention :D 

as I mentioned on [this issue](https://github.com/wazuh/wazuh-kubernetes/issues/659) I already got the 4.7.3, as its the latest available on the [hub listing](https://hub.docker.com/u/wazuh), testing to pull without tag or with 5.0.0 on another host also do not work, as with my version without tags it tries to point to :latest 

I manage to follow the steps to deploy 4.7.3, also created the storage class before as I need only 2 replicas and the actual one do not have any parameter, and get the default values of longhorn (in my case 3)

I want to get also the dashboard to be available outside, as I have a domain and proper certificates, but with lots of combinations I get 502 trying to access the domain ingress I created.

will be awesome to have included on the docs how to setup wazuh on premisses as well, as its a security feature to have it hosted in our infra for the moment.

**Comment by vcerenu** [comment]
About the pull images, if you are using Minikube or something like this, You need to upload the image into the cluster:

https://octopus.com/blog/local-images-minikube

**Comment by crlsgms** [comment]
> About the pull images, if you are using Minikube or something like this, You need to upload the image into the cluster:
> 
> https://octopus.com/blog/local-images-minikube

for the moment I'm using harbor, will add a proxy to busybox and wazuh-* 

is there a comprehensive list of image / registries needed, or I can use the image: grep

> wazuh/indexer_stack/wazuh-indexer/cluster/indexer-sts.yaml:          image: busybox
> wazuh/indexer_stack/wazuh-indexer/cluster/indexer-sts.yaml:          image: busybox
> wazuh/indexer_stack/wazuh-indexer/cluster/indexer-sts.yaml:          image: 'wazuh/wazuh-indexer:4.7.3'
> wazuh/indexer_stack/wazuh-dashboard/dashboard-deploy.yaml:          image: 'wazuh/wazuh-dashboard:4.7.3'
> wazuh/wazuh_managers/wazuh-master-sts.yaml:          image: 'wazuh/wazuh-manager:4.7.3'
> wazuh/wazuh_managers/wazuh-worker-sts.yaml:          image: 'wazuh/wazuh-manager:4.7.3'

**Comment by vcerenu** [comment]
> for the moment I'm using harbor, will add a proxy to busybox and wazuh-*
> 
> is there a comprehensive list of image / registries needed, or I can use the image: grep below?
> 
> > wazuh/indexer_stack/wazuh-indexer/cluster/indexer-sts.yaml:          image: busybox
> > wazuh/indexer_stack/wazuh-indexer/cluster/indexer-sts.yaml:          image: busybox
> > wazuh/indexer_stack/wazuh-indexer/cluster/indexer-sts.yaml:          image: 'wazuh/wazuh-indexer:4.7.3'
> > wazuh/indexer_stack/wazuh-dashboard/dashboard-deploy.yaml:          image: 'wazuh/wazuh-dashboard:4.7.3'
> > wazuh/wazuh_managers/wazuh-master-sts.yaml:          image: 'wazuh/wazuh-manager:4.7.3'
> > wazuh/wazuh_managers/wazuh-worker-sts.yaml:          image: 'wazuh/wazuh-manager:4.7.3'

Yes, with those images you can deploy the cluster without problems, we do not use others besides those.
